components/analog-devices/ADM7150.stanza

```diff
@@ -78,7 +81,7 @@ public pcb-module module (v-out:Double) :
   cap-strap(ps.vout, gnd, 10.0e-6)
   cap-strap(ps.vreg, gnd, 10.0e-6)
   cap-strap(ps.byp,  gnd, 1.0e-6)
-  cap-strap(ps.byp,  gnd, 100.0e-6)
+  cap-strap(ps.byp,  gnd, 47.0e-6)
```

Comment:
I'm curious -- does this indicate that a part's requirements have changed, or that the numbers were incorrect?

Reply:
The part has a range of acceptable values - 100uF was pretty extreme, 47uF is more reasonable.